cypress/integration/core-opensearch-dashboards/opensearch-dashboards/workspace-plugin/workspace_overview.spec.js

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { MiscUtils } from '@opensearch-dashboards-test/opensearch-dashboards-test-library';
+
+const miscUtils = new MiscUtils(cy);
+const currentDate = Date.now();
+const workspaceName = 'foo-workspace' + currentDate;
+const workspaceDescription = 'foo-workspace-desc' + currentDate;
+
+if (Cypress.env('WORKSPACE_ENABLED')) {
+  let workspaceId = '';
+
+  describe('Workspace overview', () => {
+    before(() => {
+      cy.createWorkspace({
+        workspaceName,
+        description: workspaceDescription,
+        features: [
+          'management',
+          'discover',
+          'workspace_overview',
+          'workspace_update',
+          'dashboards',
+          'visualize',
+        ],
+      }).then((id) => {
+        workspaceId = id;
+      });
+      cy.log(`workspace ${workspaceName} create successfully`);
+    });
+
+    after(() => {
+      // cy.deleteWorkspaceById(workspaceId);
```

Comment:
Do we still need this lien?

Reply:
Yes, this depends on https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6669 to be merged, otherwise the page just stuck and other test will not continue to run. Since the PR been merged, i will enable the clean up.